packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -59,6 +59,7 @@ export function ListToVirtualizedList(props) {
 		<VirtualizedList
 			id={props.id}
 			collection={props.items}
+			getComponent={props.getComponent}
```

Comment:
It's a pain to write/follow/maintain as this is passed through 
List > ListToVirtualizedList > VirtualizedList > RendererSelector > ListTable/ListGrid > gridRow > CellTitle > CellTitleActions

So 7 levels that don't need it.

What I propose : 
* remove all that
* make List accept customCellDictionary. List container will inject some customCells (more info in following points)
* ListToVirtualizedList : takes custom cell first (allow to override default ones)
* List container will connect CellTitle and pass the connected version in the custom cell dictionary. It is connected so the component will receive the getComponent.
* Make CellTitle component support getComponent, passing it to CellTitleActions that will use it.

As a general rule, do not connect a top level component and pass data through too many levels. Connect what need to be connected.